### ISSUE TITLE
Polimorph Relation Schema

### ISSUE THREAD

**Issue by FebriTahta** [prisma/prisma#25690]
## Problem
When designing a database, I thought of this relational case, where 1 column can represent 2 relations originating from 2 tables.
just like when designing dynamic menus and submenus, the urls are separated into their own table.
or when designing content, which can come from a menu or submenu

## Suggested solution

I hope Prisma can provide an ORM and support a relationship model like this, where I'm sure many users have previously used relationships like this in their database design.

## Additional context

![image](https://github.com/user-attachments/assets/a6f3f7fb-d3ee-44d6-9ab1-f44c4af13176)

**Comment by aqrln** [comment]
Am I understanding you correctly that this feature request describes what you need: https://github.com/prisma/prisma/issues/1644?

**Comment by FebriTahta** [comment]
Wow, it turns out it's already from 2020. is it available in the current version?

**Comment by aqrln** [comment]
No, but if https://github.com/prisma/prisma/issues/1644 and https://github.com/prisma/prisma/issues/2505 describe exactly what you need, please upvote those because we take upvotes into account when prioritizing.
